execution/executor/src/mock_vm/mock_vm_test.rs

```diff
@@ -70,7 +73,9 @@ fn test_mock_vm_same_sender() {
     let sender = gen_address(1);
     let mut txns = vec![];
     for _i in 0..10 {
-        txns.push(encode_mint_transaction(sender, amount));
+        txns.push(Transaction::UserTransaction(encode_mint_transaction(
```

Comment:
I didn't look into the code but maybe you can change `encode_mint_transaction` to return `Transaction`.

Reply:
That would require a lot of unrelated changes in other parts of the code base. (That's why I put in `wrap_user_txn` in executor tests. I imagine it'll be better to convert `encode_mint_transaction` and remove `wrap_user_txn` in one diff after `Transaction` is more widely used across the codebase.

Reply:
Yea I tried that and found a bit of a mess over there.

Reply:
Okay, then let's just make sure the style of these two tests are consistent. Maybe change line 40-42 like this?